golink.go

```diff
@@ -431,6 +433,9 @@ type expandEnv struct {
 	// Path is the remaining path after short name.  For example, in
 	// "http://go/who/amelie", Path is "amelie".
 	Path string
+
+	// User is the current user, if any.
```

Comment:
Can you add an example value in the comment? Prob should add to the [help.html](https://github.com/tailscale/golink/blob/main/tmpl/help.html) too.

Reply:
done